_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -0,0 +1,76 @@
+
+VSM is a method used to map value from idea to customer; looking at the processes, pain points, latency and actors involved that may impede or support the flow of value through a system. It is similar to a user journey map except the focus is on software. Read more about VSM from [Patrick Anderson](https://blog.tasktop.com/did-your-organization-wake-up-as-a-software-company-consider-value-stream-management-vsm/).
+
+There is value in designers being part of a VSM workshop. They have greater context around how the current system operates, the constraints, gaps and opportunities and how that is feeding into poor user experience. For example, with a recent client looking at AI/ML, we did not have a complete understanding of how the AI/ML worked, which meant that when we created new screens we made assumptions about the flow. It was not until we re-engaged the client that we had more context. This lack of knowledge was a constraint; it slowed us down. Now VSM is not integral to DevOps, however, VSM provides visibility into the constraints and gaps within a system and this visibility is integral to the value of DevOps. Mapping value from idea to customer provides a designer with more context on the current system and an idea of future state. This context coupled with user research leads to data driven decisions.
+<br /><br />
+
+## Why UX and DevOps
+
+DevOps without UX is possible however your return on investment and delivery success is impeded and we see this with website latency costing $2.6 billion ([Design Resources ](https://techjury.net/blog/user-experience-stats/#gref)). I would not recommend it. It is all well and good to accelerate technology delivery but that value is lost if there is no clear strategy and no understanding of users.
+
+Understanding users ensures that the flow of value from DevOps delivers a competitive advantage. Performance issues contribute to 90% of users leaving an app and functionality and design issues cause 86% of users to delete or uninstall an app ([Ewelina Luszczek](https://medium.com/@ewelina.luszczek/what-is-the-business-value-of-ux-design-affc3a1e370b)). This is not a successful deployment. Instead, when UX works together with DevOps, the inherent value of UX is unlocked providing clearer strategy and direction to DevOps. This is the competitive advantage.
+<br /><br />
+
+## Wrapping Up
+
+The growth and evolution of UX is spurred on by DevOps ways of working and likewise the success of technology delivery is through the competitive advantage of UX. There is immense value in understanding users, customers and key stakeholders needs and this value is tapped into when it enables flow through a system.
```

Comment:
Your wrap up feels weak.

What is the point of your article?  
What is your call to action?

You also need to a hook to our contact us page.